packages/admin/cms-admin/src/dam/DamTable.tsx

```diff
@@ -174,3 +177,17 @@ export const DamTable = ({ ...props }: DamTableProps): React.ReactElement => {
         </Stack>
     );
 };
+
+const NumberSelectedChip = styled("div")`
```

Comment:
This component is the same between the header and the footer, right? Could you move them into an extra file and use it in both?